src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -754,6 +755,14 @@ export default class ReimbursementRequestService {
       }
     });
 
+    const recipientSettings = await prisma.user_Settings.findUnique({
+      where: { userId: reimbursementRequest.recipientId }
+    });
+
+    if (!recipientSettings) throw new NotFoundException('Reimbursement Request', reimbursementRequestId);
+
+    sendReimburseDeniedNotificationToUser(recipientSettings?.slackId, reimbursementRequestId);
```

Comment:
you shouldn't need a ? next to the recipientSettings. That means it might be undefined, and we're explicitly checking that it's not on line 762, so we don't need it.